crates/router/src/core/webhooks/webhook_events.rs

```diff
@@ -72,6 +78,32 @@ pub async fn list_initial_delivery_attempts(
                 _ => None,
             };
 
+            let now = common_utils::date_time::now();
+            let past_90_days_ago = now - time::Duration::days(EVENT_DELIVERY_LIMIT_DAYS);
```

Comment:
Remove the duplicate code added towards the beginning of the function, and apply the suggestions here.